google-cloud-firestore/src/main/java/com/google/cloud/firestore/AggregateQuery.java

```diff
@@ -185,6 +229,66 @@ private boolean shouldRetry(Throwable throwable) {
     public void onComplete() {}
   }
 
+  ApiFuture<QueryProfileInfo<AggregateQuerySnapshot>> getAggregateQueryProfileInfo(
+      QueryMode queryMode) {
+    RunAggregationQueryRequest request = toProto().toBuilder().setMode(queryMode).build();
+    final SettableApiFuture<QueryProfileInfo<AggregateQuerySnapshot>> result =
+        SettableApiFuture.create();
+
+    ResponseObserver<RunAggregationQueryResponse> observer =
+        new ResponseObserver<RunAggregationQueryResponse>() {
+          Timestamp readTime;
+          Map<String, Value> aggregateFieldsMap = new HashMap<>();
```

Comment:
Map is never mutated. We should just use a empty map constant.

```suggestion
          Map<String, Value> aggregateFieldsMap = Collections.emptyMap();
```